### PR TITLE
add "copy link" query sharing option

### PR DESCRIPTION
**Pull request by justincorrigible**

### CODE REVIEW

File: components/theme/icons/clipboard.tsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+ *  ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ *
+ */
+
+import { ReactElement } from 'react';
+import { css } from '@emotion/core';
+
+import { IconProps } from './types';
+
+const ClipboardIcon = ({ fill = '#28519D', size = '18px', style }: IconProps): ReactElement => {
```
Comment (justincorrigible) [inline components/theme/icons/clipboard.tsx:27]: New icon

File: components/theme/icons/index.tsx [hunk 2]
```diff
@@ -35,6 +35,7 @@ import Spinner from './spinner';
 import Warning from './warning';
 
 export { default as BinIcon } from './bin';
+export { default as ClipboardIcon } from './clipboard';
```
Comment (justincorrigible) [inline components/theme/icons/index.tsx:38]: ...

File: components/pages/explorer/QueryActions/ShareButton.tsx [hunk 3]
```diff
@@ -24,8 +46,47 @@ const ShareButton = ({
   title?: string;
 }) => {
   const theme: DefaultTheme = useTheme();
+  const copyLink = useCallback(() => navigator.clipboard.writeText(queryUrl), [queryUrl]);
 
   const shareChannels = [
+    <Button
```
Comment (justincorrigible) [inline components/pages/explorer/QueryActions/ShareButton.tsx:52]: this button is not the uikit's that allows variations. 
will abstract it into a reusable component if we ever need another faceless button

File: components/pages/explorer/QueryActions/ShareButton.tsx [hunk 4]
```diff
@@ -24,8 +46,47 @@ const ShareButton = ({
...
+
+        > span {
+          align-items: center;
+          display: flex;
+          flex-wrap: nowrap;
+        }
+      `}
+      onClick={copyLink}
+    >
+      <div
```
Comment (justincorrigible) [inline components/pages/explorer/QueryActions/ShareButton.tsx:73]: gives the icon a background square

File: components/Dropdown/index.tsx [hunk 5]
```diff
@@ -123,7 +123,7 @@ const Dropdown = ({
               margin: 0;
               max-width: 300px;
               min-width: 100%;
-              padding: 0;
+              padding: 2px 0 1px;
```
Comment (justincorrigible) [inline components/Dropdown/index.tsx:126]: fine-tuning the dropdown style to compensate for fonts' optical effects.

**Review by anncatton — APPROVED** [review]
🚀 ©️

**Pull request merged**